cmd/bosun/sched/check.go

```diff
@@ -284,8 +311,7 @@ Loop:
 		}
 		event := rh.Events[ak]
 		if event == nil {
-			event = new(Event)
```

Comment:
We need this line because `event` is used below, and it'll still be `nil` if not set here.